logistics/scripts/modules/settings/settings-management.psm1

```diff
@@ -312,21 +321,40 @@ function New-DevelopmentAppSettings([hashtable] $Settings = @{ }) {
         $developmentSettingsForEngine = (Get-DefaultDevelopmentSettingsByEngine)[$Settings.ApiSettings.Engine]
         $developmentSettingsForEngine = Add-ApplicationNameToConnectionStrings $developmentSettingsForEngine $project
 
-        $newDevelopmentSettings = Merge-Hashtables $developmentSettingsByProject[$project], $developmentSettingsForEngine, $Settings
+        $newDevelopmentSettings = Merge-Hashtables $developmentSettingsByProject[$project], $developmentSettingsForEngine
 
         $newDevelopmentSettings = Add-TestHarnessSpecificAppSettings $newDevelopmentSettings $project
 
+        $newDevelopmentSettings = Merge-Hashtables $newDevelopmentSettings, (Get-CredentialSettingsByProject)[$project], $Settings
+
         $projectPath = Get-RepositoryResolvedPath $Project
 
         $newDevelopmentSettingsPath = Join-Path $projectPath "appsettings.development.json"
         New-JsonFile $newDevelopmentSettingsPath $newDevelopmentSettings -Overwrite
         $newSettingsFiles += $newDevelopmentSettingsPath
 
         $credentialSettings = Merge-Hashtables @{ }, (Get-CredentialSettingsByProject)[$project]
-        $newUserSettingsPath = Join-Path $projectPath "appsettings.user.json"
-        New-JsonFile $newUserSettingsPath $credentialSettings
-        $newSettingsFiles += $newUserSettingsPath
+        $userSecrets += ($credentialSettings | ConvertTo-Json -Depth 10 | dotnet user-secrets set --project $projectPath --id (Get-UserSecretsIdByProject).$project)
     }
 
-    return $newSettingsFiles
+    return ($newSettingsFiles, $userSecrets)
 }
+
+function Display-UserSecretsList() {
```

Comment:
This function is outputting warnings everytime it is imported
```
WARNING: The names of some imported commands from the module 'settings-management' include unapproved verbs that might 
14:05:31
  make them less discoverable. To find the commands with unapproved verbs, run the Import-Module command again with the 
14:05:31
  Verbose parameter. For a list of approved verbs, type Get-Verb.
```

Reply:
I fixed this warnings